tgui/packages/tgui/interfaces/BodyScanner.js

```diff
@@ -5,29 +5,28 @@ import { AnimatedNumber, Box, Button, Flex, Icon, LabeledList, ProgressBar, Sect
 import { Window } from "../layouts";
 
 const stats = [
-  ['good', 'Alive'],
-  ['average', 'Critical'],
-  ['bad', 'DEAD'],
+  ['good', 'Жив'],
+  ['average', 'Критическое состояние'],
+  ['bad', 'ТРУП'],
 ];
 
 const abnormalities = [
-  ['hasBorer', 'bad', 'Large growth detected in frontal lobe,'
-    + ' possibly cancerous. Surgical removal is recommended.'],
-  ['hasVirus', 'bad', 'Viral pathogen detected in blood stream.'],
-  ['blind', 'average', 'Cataracts detected.'],
-  ['colourblind', 'average', 'Photoreceptor abnormalities detected.'],
-  ['nearsighted', 'average', 'Retinal misalignment detected.'],
+  ['hasBorer', 'bad', 'Во фронтальной коре обнаружен крупный нарост. Потенциальная онкология. Рекомендуется хирургическое вмешательство.'],
+  ['hasVirus', 'bad', 'В крови обнаружен вирусный патоген.'],
+  ['blind', 'average', 'Обнаружена катаракта.'],
+  ['colourblind', 'average', 'Обнаружена аномалия фоторецепторов.'],
+  ['nearsighted', 'average', 'Обнаружено смещение зрительной сетчатки.'],
 ];
 
 const damages = [
-  ['Respiratory', 'oxyLoss'],
-  ['Brain', 'brainLoss'],
-  ['Toxin', 'toxLoss'],
-  ['Radioactive', 'radLoss'],
-  ['Brute', 'bruteLoss'],
-  ['Genetic', 'cloneLoss'],
-  ['Burn', 'fireLoss'],
-  ['Paralysis', 'paralysis'],
+  ['Асфиксия', 'oxyLoss'],
+  ['Мозг', 'brainLoss'],
+  ['Интоксикация', 'toxLoss'],
+  ['Радиационные', 'radLoss'],
```

Comment:
Понравилось такое название из печатных сканов. Само по себе это не повреждение - тем же нуклеациям от этого по-боку. (Если бы работал механ, наоборот хилится должны бы были)
```suggestion
  ['Уровень радиации', 'radLoss'],
```